tools/reviewer/aa/commands/DiffCommand.java

```diff
@@ -178,5 +184,38 @@ public boolean run(String[] args) {
     codeReviewBlockingStub.createDiff(request);
     return true;
   }
+
+  private void addGithubRepo(Diff.Builder diffBuilder) {
```

Comment:
I think - since we're actually going to create PRs in these repos, we should only add repos which have changes in them. So how about you check for that instead?
You can add a method to `GitRepo` that checks if there are any changes from `master`. Something like `hasChages()`. It would check that `getCommits()` is not empty, or that `getUncommittedFiles()` is not empty.